dist/plugins/angular.min.js.map

```diff
@@ -0,0 +1 @@
+{"version":3,"sources":["../../angular.js"],"names":["f","exports","module","define","amd","g","window","global","self","this","Raven","Plugins","Angular","e","t","n","r","s","o","u","a","require","i","Error","code","l","call","length",1,"_dereq_","angularPlugin","angular","RavenProvider","$get","$window","ExceptionHandlerProvider","$provide","decorator","exceptionHandler","R","$delegate","ex","cause","captureException","extra","provider","config","setDataCallback","data","exception","values","matches","angularPattern","exec","value","type","message","angularDocs","substr"],"mappings":";CAYA,SAAUA,GAAG,GAAoB,gBAAVC,UAAoC,mBAATC,QAAsBA,OAAOD,QAAQD,QAAS,IAAmB,kBAATG,SAAqBA,OAAOC,IAAKD,UAAUH,OAAO,CAAC,GAAIK,EAAkCA,GAAb,mBAATC,QAAwBA,OAA+B,mBAATC,QAAwBA,OAA6B,mBAAPC,MAAsBA,KAAYC,KAAKJ,EAAGA,EAAEK,QAAQL,EAAEK,UAAaL,EAAGA,EAAEM,UAAUN,EAAEM,YAAeN,EAAEO,QAAUZ,MAAO,WAAqC,MAAO,SAAUa,GAAEC,EAAEC,EAAEC,GAAG,QAASC,GAAEC,EAAEC,GAAG,IAAIJ,EAAEG,GAAG,CAAC,IAAIJ,EAAEI,GAAG,CAAC,GAAIE,GAAkB,kBAATC,UAAqBA,OAAQ,KAAIF,GAAGC,EAAE,MAAOA,GAAEF,GAAE,EAAI,IAAGI,EAAE,MAAOA,GAAEJ,GAAE,EAAI,IAAIlB,GAAE,GAAIuB,OAAM,uBAAuBL,EAAE,IAAK,MAAMlB,GAAEwB,KAAK,mBAAmBxB,EAAE,GAAIyB,GAAEV,EAAEG,IAAIjB,WAAYa,GAAEI,GAAG,GAAGQ,KAAKD,EAAExB,QAAQ,SAASY,GAAG,GAAIE,GAAED,EAAEI,GAAG,GAAGL,EAAG,OAAOI,GAAEF,EAAEA,EAAEF,IAAIY,EAAEA,EAAExB,QAAQY,EAAEC,EAAEC,EAAEC,GAAG,MAAOD,GAAEG,GAAGjB,QAAkD,IAAI,GAA1CqB,GAAkB,kBAATD,UAAqBA,QAAgBH,EAAE,EAAEA,EAAEF,EAAEW,OAAOT,IAAID,EAAED,EAAEE,GAAI,OAAOD,KAAKW,GAAG,SAASC,EAAQ3B,EAAOD,GAMl4B,YAKA,SAAS6B,GAAcpB,EAAOqB,GAK1B,QAASC,KACLvB,KAAKwB,MAAQ,UAAW,SAASC,GAC7B,MAAOxB,KAIf,QAASyB,GAAyBC,GAC9BA,EAASC,UAAU,qBACd,QAAS,YAAaC,IAG/B,QAASA,GAAiBC,EAAGC,GACzB,MAAO,UAAUC,EAAIC,GACjBH,EAAEI,iBAAiBF,GACfG,OAASF,MAAOA,KAEpBF,EAAUC,EAAIC,IApBtBX,EAAUA,GAAWzB,OAAOyB,QAEvBA,IAsBLA,EAAQ7B,OAAO,cACV2C,SAAS,QAAUb,GACnBc,QAAQ,WAAYX,IAEzBzB,EAAMqC,gBAAgB,SAASC,GAE3B,GAAIC,GAAYD,EAAKC,SACrB,IAAIA,EAAW,CACXA,EAAYA,EAAUC,OAAO,EAC7B,IAAIC,GAAUC,EAAeC,KAAKJ,EAAUK,MAExCH,KAEAF,EAAUM,KAAOJ,EAAQ,GACzBF,EAAUK,MAAQH,EAAQ,GAC1BH,EAAKQ,QAAUP,EAAUM,KAAO,KAAON,EAAUK,MAEjDN,EAAKJ,MAAMa,YAAcN,EAAQ,GAAGO,OAAO,EAAG,UA5C9D,GAAIN,GAAiB,2DAkDrBlD,GAAOD,QAAU6B,YAEN,IAAI","file":"../../angular.min.js"}
```

Comment:
These `sources` are pointing to the wrong files. So we need to fix sourcemap generation.